code/src/components/DetailSite.js

```diff
@@ -0,0 +1,98 @@
+import React, { useState, useEffect } from "react";
+import { useParams } from "react-router";
```

Comment:
you can use "react-router-dom" instead, might make the build smaller. react-router is bigger
https://stackoverflow.com/questions/42684809/react-router-vs-react-router-dom-when-to-use-one-or-the-other